frontend/src/concepts/pipelines/content/createRun/RunPage.tsx

```diff
@@ -54,11 +54,16 @@ const RunPage: React.FC<RunPageProps> = ({ cloneRun, contextPath, testId }) => {
 
   const isExperimentsAvailable = useIsAreaAvailable(SupportedArea.PIPELINE_EXPERIMENTS).status;
 
-  const sections = isExperimentsAvailable
-    ? Object.values(CreateRunPageSections)
-    : Object.values(CreateRunPageSections).filter(
-        (section) => section !== CreateRunPageSections.EXPERIMENT,
-      );
+  const jumpToSections = Object.values(CreateRunPageSections).filter((section) => {
+    if (
+      (section === CreateRunPageSections.EXPERIMENT && !isExperimentsAvailable) ||
```

Comment:
a nit, but could be condensed down into a single boolean instead of an if returning true or false